Solutions/InsiderRiskManagement/Analytic Rules/InsiderM365IRMAlertObserved.yaml

```diff
@@ -0,0 +1,38 @@
+id: 69660e65-0e5c-4700-8b99-5caf59786606
+name: (Private Preview) Insider Risk - Microsoft 365 Insider Risk Management Alert Observed
```

Comment:
Do we want to put Private Preview here or just Preview?  We have traditionally used Preview for things like TI. :)